deploy/crds/ssp.kubevirt.io_v1_ssp_cr.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: ssp.kubevirt.io/v1
+kind: SSP
+metadata:
+  name: example-ssp
```

Comment:
shouldn't this be changed to something other then example-ssp?

Reply:
Changed to `kubevirt-ssp`.